web/firefoxcom.js

```diff
@@ -285,6 +285,47 @@ class MozL10n {
   window.addEventListener("editingaction", handleEvent);
 })();
 
+if (
+  typeof PDFJSDev === "undefined"
+    ? window.isGECKOVIEW
+    : PDFJSDev.test("GECKOVIEW")
+) {
```

Comment:
Given that this file isn't used during development, note that check at the top of the file, this could be simplified as follows.
```suggestion
if (PDFJSDev.test("GECKOVIEW")) {
```